addons/point_of_sale/models/pos_payment.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, fields, models
+
+
+class PosPayment(models.Model):
+    """ This model will be used to prevent using account.payment for each pos.order.
+        The pos.payment for each pos.session will be combined to generate one (or more if multiple types)
+        account.payment. This will reduce the number of account.payment, account.move and account.move.line
+        that will be created when closing a pos session.
+    """
+
+    _name = "pos.payment"
+    _description = "Point of Sale Payments"
+    _order = "id desc"
+
+    name = fields.Char(readonly=True, copy=False)
+    pos_order_id = fields.Many2one(comodel_name='pos.order', string='Pos Order')
```

Comment:
rename 'order_id', remove comodel everywhere it is written

Reply:
I will rename them. I will also rename those fields that I put pos_ at the beginning.